package.json

```diff
@@ -22,7 +22,8 @@
     "release": "np --no-2fa"
   },
   "peerDependencies": {
-    "react": ">=16.8"
+    "react": ">=16.8",
+    "react-dom": ">=16.8"
```

Comment:
We're going to need this. Idk if this deserves a major/minor bump. Since this package technically only works with react-dom anyway.